packages/mds-ingest-service/repository/migrations/1652752567678-AddAccessibilityOptionsColumnToEventAnnotations.ts

```diff
@@ -0,0 +1,16 @@
+import type { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class AddAccessibilityOptionsColumnToEventAnnotations1652752567678 implements MigrationInterface {
+  name = 'AddAccessibilityOptionsColumnToEventAnnotations1652752567678'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "event_annotations" ADD "accessibility_options" character varying(255) array NOT NULL DEFAULT '{}'`
+    )
+    await queryRunner.query(`ALTER TABLE "event_annotations" ALTER COLUMN "accessibility_options" DROP DEFAULT`)
```

Comment:
@stanleychukwuocha should this be a constant-time operation because we're defaulting to a static value, or will this be super slow potentially?

Reply:
@PeterSP we don't actually want this to default in typical operations, it should be a required property when writing to this table. Having it at all is to avoid a costly backfill operation (which won't be necessary for older rows, as at the moment we don't have anything with accessibility options registered).

Reply:
Ah, so the initial default is simply for backfill.

Reply:
i think this should be fine. If there is any concerns, I can add that ahead of the deployment so it no ops.